tools/devops/automation/templates/build/build.yml

```diff
@@ -136,82 +138,87 @@ steps:
     # a branch in origin
     
     if ($buildReason -eq "PullRequest" -or (($buildReason -eq "Manual") -and ($buildSourceBranchName -eq "merge")) ) {
-      Write-Host '##vso[task.setvariable variable=IsPR;isOutput=true]False'
+      $configVars.Add("IsPR", "True")
```

Comment:
The change of the value is literally the reason I created this PR, This is a PR and was set to false, probably done in a bad merge.